src/routes/create-dictionary.svelte

```diff
@@ -95,6 +96,7 @@
         //@ts-ignore
         termsAgreement: serverTimestamp(),
       });
+      addDictionaryManagePermission($user, url);
```

Comment:
@Danble Please see my final solution to just do the adjustment in the user update operation already happening. Calling this function would cause the user's info to be set in the dictionaries managing collection twice. Also, make sure to use `await` when calling an async function.